runtime/appruntime/exported/config/config.go

```diff
@@ -44,13 +44,32 @@ type Runtime struct {
 	RedisServers    []*RedisServer          `json:"redis_servers,omitempty"`
 	RedisDatabases  []*RedisDatabase        `json:"redis_databases,omitempty"`
 	Metrics         *Metrics                `json:"metrics,omitempty"`
+	Services        []*Service              `json:"services,omitempty"`
```

Comment:
Would it make sense to call this "service discovery" instead? Should it be a `map[string]Service` instead? (Also probably not worth having pointers)

Reply:
👍